### ISSUE TITLE
Search inside Playlist

### ISSUE THREAD

**Issue by dPandl** [maxrave-dev/SimpMusic#182]
My playlists are very large and sometimes I want to find a specific song that I know is in the playlist

With Spotify you can search in playlists and find songs more quickly. The same function should also be available here.

Example from Spotify: 
![IMG_20231129_001418](https://github.com/maxrave-dev/SimpMusic/assets/35048273/f7160038-09ae-425c-b057-c25e56436a43)

**Comment by vishalscodes** [comment]
Yes, that feature should be available as it helps in deleting unwanted songs quickly.
